airflow-core.yaml

```diff
@@ -0,0 +1,149 @@
+package:
+  name: airflow-core
+  version: 2.10.4
+  epoch: 0
+  description: Minimal Airflow core - Platform to programmatically author, schedule, and monitor workflows
+  dependencies:
+    runtime:
+      - bash
+      - busybox
+      - coreutils
+      - curl
+      - cyrus-sasl
+      - cyrus-sasl-dev
+      - findutils
+      - glibc-locale-en
+      - glibc-locales
+      - gnupg-utils
+      - gzip
+      - mariadb-dev
+      - mysql
+      - openssl
+      - postgresql-client
+      - postgresql-dev
+      - python-3.12
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - gcc
+      - glibc-dev
+      - mariadb-connector-c-dev
+      - mariadb-dev
+      - openssf-compiler-options
+      - pkgconf-dev
+      - postgresql-dev
+      - py3.12-build-bin
+      - py3.12-pip
+      - python-3.12
+      - python-3.12-dev
+      - wolfi-base
+      - zlib-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/apache/airflow
+      tag: ${{package.version}}
+      expected-commit: c083e456fa02c6cb32cdbe0c9ed3c3b2380beccd
+
+  - runs: |
+      # Set required build environment variables
+      export SOURCE_DATE_EPOCH=315532800
+      export MYSQLCLIENT_CFLAGS=$(mysql_config --cflags)
+
+      # Install airflow core, the upstream sets this as directory
+      export PYTHONUSERBASE=${{targets.destdir}}/home/airflow/.local
+      pip install --no-cache-dir --user apache-airflow[core]==${{package.version}}
+
+      #GHSA-8w49-h785-mj3c/GHSA-8495-4g3g-x7pr/GHSA-27mf-ghqm-j3j8 fixes
+      pip install --no-cache-dir --user aiohttp>=3.10.11 tornado>=6.4.2
+      pip install --no-cache-dir --user --force-reinstall packaging statsd
+
+  - runs: |
```

Comment:
If we aren't breaking things into labelled steps, I'd prefer we just used whitespace to separate things logically rather than invoking run again

Reply:
done, thanks RJ